### PR TITLE
Basic ui

### PR DESCRIPTION
**Pull request by Mithi-code**
### OOP school library: add basic UI

- Build interactive console apps.
- Create an app.rb file that will serve as your console app entry-point.
- In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file.
- Run a program using the command line.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    case parent_permission
+    when 'n'
+      Student.new(age, 'classroom', name, parent_permission: false)
+      @persons << student
+      puts 'Student doesnt have parent permission, cant rent books'
+    when 'y'
+      student = Student.new(age, 'classroom', name, parent_permission: false)
+      @persons << student
+      puts 'Student created successfully'
+    end
```
Comment (rloterh) [inline app.rb:69]: - [ ] Your app is throwing an error when option `y` was entered for _has parent permission_ option. Kindly ensure to improve upon this aspect.

**Error:**  
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    puts 'Create a new teacher'
+    print 'Enter teacher age: '
+    age = gets.chomp.to_i
+    print 'Enter teacher name: '
+    name = gets.chomp
+    print 'Enter teacher specialization: '
+    specialization = gets.chomp
+    teacher = Teacher.new(age, specialization, name)
+    @persons << teacher
+    puts 'Teacher created successfully'
```
Comment (rloterh) [inline app.rb:82]: - [ ] Your app is throwing an error when a text was entered for _specialization_ of a teacher option. Kindly ensure to improve upon this aspect.

**Error:**  
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mithi, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @Mithi-code,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**